### ISSUE TITLE
[BUG] Flight control not showing up

### ISSUE THREAD

**Issue by Santvarg** [DarkholmeTenk/TardisModIssues#532]
About half of the time, the blue indicator does not show up, causing an explosion because I can't see which flight control I'm supposed to use.
Sometimes I do see it flash blue for a tick or two(not an exaggeration) after not showing up when it should, before it explodes, even though I'm only at the 2nd speed setting.

This happens whether drifting or not, and I am using the latest version.(1.7.10-0.994-124-DS)
Singleplayer.

I am using optifine, if that would cause it. ( Edit: Not optifine, uninstalled with no effect)
If needed I can post all the used mods.

**Comment by FoxPotato** [comment]
what speed is your TARDIS on? like the speed lever, iirc that affects how long you see the blue highlight

**Comment by Santvarg** [comment]
It happens on all speeds I've used, from the lever being on the slowest setting, to as high as I feel comfortable flying (considering my predicament), which is the 2nd speed setting from the slowest. IE: set speed to 0.0, click twice.

**Comment by FoxPotato** [comment]
please post the mods yes, and maybe the logs after flying a bit and seeing some levers.

**Comment by Santvarg** [comment]
Running: MC 1.7.10, obv; with Liteloader and forge 10.13.4.1492
Project Red 4.7.0-12.95
iChunUtil 4.0.0
Morph 0.9.2
Project E PE1.10.0
darkcore 0.3-86
DragonMounts r41
buildcraft 7.1.16
immibis core 59.1.4
Inventory Tweaks 1.59-dev-152
Modular Powersuits-0.11.0.301
Numina 0.4.0.131
Ic2 EXP 2.2.822 
Advances MachinesAS
IC2 Nuclear Control 2.4.0a
Adv Repulsion Systems 59.0.4
Advanced Solar Panel 3.5.1
Tardis Mod, 0.994-124-DS
Galacticraft Core 3.0.12.400
Galacticraft Planets, 3.0.12.400
MicDoodleCore 3.0.12.400
Mantle 0.3.2
shipsMod 1.0.4
forestry 4.0.11.40
BiomedOPlenty 2.1.0.1462
Ironchest 6.0.62.742
WAILA 1.5.10
Spacecore 0.7.14
Useful Pets 1.3
ChickenChunks 1.3.4.16
CodeChickenCore 1.0.7.46
NEI 1.0.5.111
Voxel Map
Optifine HD-U-C1
Mystcraft 0.12.1.00
MAtmos r28b

Logs will follow in...
Soon.

**Comment by Santvarg** [comment]
I did logs, noting what I was doing. I also Pasted Failed into chat when I couldn't see one. There was one incident where there was no explosion.(Failed?)

Edit: Moved to Pastebin
http://pastebin.com/HKME72Zb

**Comment by Santvarg** [comment]
I could record what I see on my end, if it would help.

**Comment by FoxPotato** [comment]
could you perhaps post that to pastebin and not just in a comment

**Comment by Santvarg** [comment]
Yes, I will be sleeping now, I'll check back here again in 8-12 hours

**Comment by Santvarg** [comment]
Has the problem been identified?
I'm going to try to remove chickenChunks later, long shot but it's the only mod I can think of that might interfere.

**Comment by AlienXtream** [comment]
i think ive seen this but just assumed i was not seeing the button somwhere

**Comment by Santvarg** [comment]
I know sometimes the small buttons at the coord save appear white from the sides when looking at them through a console, but when I fly manually I circle around the console, always facing it, so I WILL see it, if it shows up.

**Comment by AlienXtream** [comment]
well if i were to have an uneducated guess i would think that its "highlighting" a button that isnt actually there (present in the code but not actually placed on the console)

**Comment by Santvarg** [comment]
My guess is that either:
Similar to what you said, something is keeping the chunk from updating with the information about the controls.
Or It wasn't able to properly choose the control, thus also unable to highlight nothing.

**Comment by FoxPotato** [comment]
is this on a server btw?

**Comment by Santvarg** [comment]
No, Singleplayer. Alien might be though.

**Comment by AlienXtream** [comment]
sp, but i assume the same would be true on servers

**Comment by Santvarg** [comment]
Will be trying a TARDIS only installation later, to ascertain if it's a native problem, or if it's being caused by interference. Will edit this post.

Edit: Entirely clean install, recommended forge and the newest versions of TARDIS and darkcore. ( Using the Beta versions previously, These are labeled Alpha )
No other mods, loaders, resource packs. Erased entire MC folder before reinstalling. Ran with the forge profile. Nothing new, so it's definitely the mod itself.

**Comment by Santvarg** [comment]
Has the problem been discovered?

**Comment by AlienXtream** [comment]
not to my knowledge.

**Comment by shadowbolt79** [comment]
Both myself and a friend of mine have been experiencing this issue as well.

**Comment by Santvarg** [comment]
I hope Darkholme has/will/can fix this, Manual flying becomes very expensive.

**Comment by shadowbolt79** [comment]
It is. Sometimes I'm lucky and never get my tardis completely broken; as long as you can get to the point where you can make enough chronosteel/dalakanium so you can fit the upgrade slots with explosion resistance, misclick protection, and shield boosters, you should be fine as long as you don't fly back-to-back. And never drift in the time space continuum.

**Comment by Santvarg** [comment]
I have EE2 on that installation, so it's not actually expensive, but if I didn't, I'd never be able to fly it without going into creative xD

**Comment by FoxPotato** [comment]
I can't really remember (it's been ages since ive used the mod :P) but doesn't stable flight remove the explosions? which you get at like level 5 or something like that?

**Comment by AlienXtream** [comment]
I would assume they are xp grinding. Of course if thats the case then they are doing it the hard way :p. That xp bug you found is still a thing XD

**Comment by Santvarg** [comment]
Stable flight does remove the need to mess with the buttons, but Flying manually is pretty much impossible because you can't properly... fly, what with the button errors.

Unfortunately Manual flight is also a great way to get XP, which I need because I ran out of room...

**Comment by shadowbolt79** [comment]
It's more like 'I'm flying a level 1 tardis, and it keeps crashing trying to get from point A to B because half of the controls don't light up.' Once I get the tardis leveled up, I'm fine. I don't actually miss many of the ones that actually light up. Mostly my beginnings with this bug is a race to make the shield, misclick protection, and explosion protection upgrades ASAP so my kontron crystal repair core doesn't break. Very expensive to repair.

**Comment by TravMiner123** [comment]
This maybe 4 years late but going into stabe flight and back into unstable flight, resets the mingame. So after a button is pressed and you dont see anymore using this control could reset it to have one show up.
Also spamming it seems to give insain xp since having the blue control show up gives the xp and not when its pressed

**Comment by AlienXtream** [comment]
you are right. this IS 4 years late XD. this mod has been dead for some time now. no support is being provided. if you are still one 1.7.10 thats fine but i recommend updating to 1.12.2 as it is the current mainstream modding version and has been for a while now so many "must have" mods are updated. there is also a "new" tardis mod that was inspired by this one but is doing its own thing. its bare bones right now but its making progress
